style.css

```diff
@@ -1,633 +1,619 @@
-body {
```

Comment:
why on entire file tab was added? Now I cannot see changes of file because entire file has changed.

Reply:
I think that i did it because it was simpeler and faster to attach a new classes with a whole code instead of searching where the new code has to be placed. :) (yes, bad choice..)

Reply:
nvm. I wrongly understood comment.

fixed